mail_outbound_static/models/ir_mail_server.py

```diff
@@ -46,3 +85,26 @@ def send_email(
         return super(IrMailServer, self).send_email(
             message, mail_server_id, smtp_server, *args, **kwargs
         )
+
+    @api.model
```

Comment:
perhaps we could use ormcache because this method is going to be called many times.
Should be something like this:

```
@tools.ormcache('mail_server_id', 'smtp_server', 'email_domain')
def _get_mail_sever(self, mail_server_id, smtp_server, email_domain):
```

Then on the write/create/unlink we need to call the invalidate cache as [here](https://github.com/odoo/odoo/blob/13.0/addons/mail/models/mail_message_subtype.py#L44-L55)